app/controllers/presentations_controller.rb

```diff
@@ -11,6 +11,7 @@ def show
 
   def new
     @presentation = Presentation.new
+    @event = Event.find(params[:event_id]) if params[:event_id]
```

Comment:
In the next commit though you call `@event&.id` anyway though. when you call an instance variable it is created with the initial value of nil. 

i'd either to the `find_by` or something like `@event = Event.find_by(id: params[:event_id]) || Event.new` 
that way you're always operating on an event object. 

Id also consider always building the presentation off of the event object `@event.presentations.build`